eZ/Publish/Core/FieldType/RichText/Resources/schemas/docbook/docbook.iso.sch

```diff
@@ -246,6 +246,7 @@
     <s:rule context="db:link">
       <s:assert test="not(.//db:link)">link must not occur in the descendants of link</s:assert>
       <s:assert test="not(.//db:ezlink)">ezlink must not occur in the descendants of link</s:assert>
+      <s:assert test="not(contains(@*[name()='xlink:href'], 'javascript:') or contains(@*[name()='xlink:href'], 'vbscript:'))">using scripts in links is not allowed</s:assert>
```

Comment:
> question: are the on* attributes (onclick, onfocus, ...) kept (not only on links btw)

@dpobel: I tried to inject onclick, but it seems to be removed silently when storing richtext data

> If possible this this should be changed to whitelist, hence avoiding such issues.
>
> ezurl|ezcontent|ezlocation|ezremote|..?

@andrerom: when xml gets validated it doesn't contain these yet. They are created in [RichTextStorage](https://github.com/ezsystems/ezpublish-kernel/blob/v6.7.2/eZ/Publish/Core/FieldType/RichText/RichTextStorage.php#L39-L115).
(xpath expression gets all the links that need to be converted, so except already stored `ezurl://`, etc. - at least that's how I understand this code :-) ).
I could validate it in Storage, but I'm not sure if this is the right place to throw such exceptions.

// cc @pspanja